docs/book/src/developer/providers/implementers-guide/overview.md

```diff
@@ -37,33 +37,17 @@ KUBECTL_VERSION=$(curl -sfL https://dl.k8s.io/release/stable.txt)
 curl -fLO https://dl.k8s.io/release/${KUBECTL_VERSION}/bin/linux/amd64/kubectl
 
 # Install kustomize
-OS_TYPE=linux
-FILE=kustomize_*_${OS_TYPE}_amd64.tar.gz
-curl -sf https://api.github.com/repos/kubernetes-sigs/kustomize/releases/latest |\
-  grep browser_download |\
-  grep ${OS_TYPE} |\
-  cut -d '"' -f 4 |\
-  xargs curl -f -O -L
-tar zxvf $FILE; rm -f $FILE
-sudo mv kustomize /usr/local/bin/kustomize
-sudo chmod u+x /usr/local/bin/kustomize
+curl -s "https://raw.githubusercontent.com/kubernetes-sigs/kustomize/master/hack/install_kustomize.sh"  | bash
```

Comment:
Looks good to me apart from one question - should we specify the versions of these tools we're using, or do we think the most recent is always going to be the most suitable release?

Reply:
I would prefer something best effort